helpers/fail2ban

```diff
@@ -102,6 +100,15 @@ ignoreregex =
     ynh_add_config --template="f2b_jail.conf" --destination="/etc/fail2ban/jail.d/$app.conf"
     ynh_add_config --template="f2b_filter.conf" --destination="/etc/fail2ban/filter.d/$app.conf"
 
+    # Create the folder and logfile if they doesn't exist,
+    # as fail2ban require an existing logfile before configuration
+    mkdir -p "/var/log/$app"
+    if [ ! -f "$logpath" ]; then
+        touch "$logpath"
+        chown -R "$app:$app" "$logpath"
+        chmod -R u=rwX,g=rX,o= "$logpath"
```

Comment:
```suggestion
        chown "$app:$app" "$logpath"
        chmod u=rw,g=r,o= "$logpath"
```

i'm a bit uncomfortable with the `-R` and folder-related options if `logpath` is meant to be a file ?

Reply:
ah yes, in fact I foolishly copied the @Salamandar code in https://github.com/YunoHost/example_ynh/pull/216

Reply:
I made the appropriate fix

Reply:
That is not an issue :
* X (caps) only applies to directories
* -R does not fail if it's a file

Reply:
yes but I made a mistake by indicating only the file and not the folder!
with the folder, the configuration will also be applied to the file (created just before)